conf/application.conf

```diff
@@ -149,6 +149,13 @@ microservice {
         environment = "qa"
     }
 
+    declaration-info-request {
+        host = localhost
+        port = 7502
+        context-base = "/accounts/overpaymentdeclarationdisplay/v1"
```

Comment:
Is there someway to avoid exposing these real endpoints in our application.conf? I would prefer to only have these in the actual app-config-* files. Because we code in the open we expose these unnecessarily.

Reply:
I can rename them, also rename the enpoints in cds-reimursements-stub
and put the current values into app-config base
is that ok?